data_structures/binary_tree/binary_tree_traversals.py

```diff
@@ -0,0 +1,149 @@
+class Node:
+    """
+    A Node has data variable and pointers toits left and right nodes.
+    """
+    def __init__(self,data):
+        self.left=None
+        self.right=None
+        self.data=data
+
+def preOrder(root):
+    """
+    PreOrder traversal: visit root node then its left subtree followed by right subtree.
+    """
+    if(root!=None):

```

Comment:
These ( ) are not needed and pfs/black will remove them.

Reply:
what if i don't remove them, Any problem with that?

Reply:
Yes.  Flake8 will fail your submission and because it is not green it will never be reviewed and in a week the stalebot will add a stale label to the PR and a week later the stalebot will close your PR.

Reply:
ok